BigInteger/BigInteger.cs

```diff
@@ -26,77 +28,49 @@ public class BigInteger
         1993, 1997, 1999 };
 
 
-    private uint[] data = null;            // stores bytes from the Big Integer
-    public int dataLength;                 // number of actual chars used
-
+    private uint[] data = new uint[MAX_LENGTH];             // stores bytes from the Big Integer
+    public int dataLength { get; set; }                 // number of actual chars used
```

Comment:
I made a mistake here to put `setter` there because I saw it used `public`. However, is it meant to be `private`?